pkg/reconcilermanager/controllers/rootsync_controller.go

```diff
@@ -544,8 +556,8 @@ func (r *RootSyncReconciler) mapObjectToRootSync(obj client.Object) []reconcile.
 
 	// Ignore changes from resources without the root-reconciler prefix or configsync.gke.io:root-reconciler
 	// because all the generated resources have the prefix.
-	roleBindingName := RootSyncPermissionsName()
-	if !strings.HasPrefix(objRef.Name, core.RootReconcilerPrefix) && objRef.Name != roleBindingName {
+	if !strings.HasPrefix(objRef.Name, core.RootReconcilerPrefix) &&
+		!strings.HasPrefix(objRef.Name, fmt.Sprintf("%s:%s", configsync.GroupName, core.RootReconcilerPrefix)) {
```

Comment:
Maybe we shouldn't `fmt.Sprintf` here, but rather try to depend on something like `RootSyncPermissionsName(core.RootReconcilerPrefix)` to ensure that it stays consistent if we change the naming scheme?
```suggestion
	if !strings.HasPrefix(objRef.Name, core.RootReconcilerPrefix) &&
		!strings.HasPrefix(objRef.Name, RootSyncPermissionsName(core.RootReconcilerPrefix)) {
```

Reply:
I think I'm gonna go with this, since I prefer it myself.

Reply:
Maybe we could add another const which stores this prefix value? e.g. `RootSyncClusterRoleBindingPrefix`

That way we can reuse the string and ensure consistency

Reply:
Yeah, that's not a bad idea.